scripts/globals/magic.lua

```diff
@@ -1329,6 +1356,12 @@ function calculateDuration(duration, magicSkill, spellGroup, caster, target, use
         if caster:hasStatusEffect(dsp.effect.SABOTEUR) then
             duration = duration * 2
         end
+
+        -- After Saboteur according to bg-wiki
+        if caster:getMainJob() == dsp.job.RDM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.